base/channels.jl

```diff
@@ -380,8 +380,37 @@ end
 """
     fetch(c::Channel)
 
-Wait for and get the first available item from the channel. Does not
-remove the item. `fetch` is unsupported on an unbuffered (0-size) channel.
+Waits for and returns (with removing) the first available item from the `Channel`.
+Note: `fetch` is unsupported on an unbuffered (0-size) `Channel`.
+
+# Examples
+
+Buffered channel:
+```jldoctest
+julia> c = Channel(3) do ch
+    foreach(i -> put!(ch, i), 1:4)
+end;
+
+julia> fetch(c)
+1
```

Comment:
I could probably show that after fetching the item is not removed?

Reply:
Sure, if you want to add a line with `collect`

Reply:
added